ci.sh

```diff
@@ -26,7 +26,7 @@ build_path=generated/$OS_NAME/release/$MODEL
 if [ "$OS_NAME" == "linux" ]; then
     mkdir -p linker
     rm -f linker/ld
-    ln -s /usr/bin/ld.gold linker/ld
+    ln -s /usr/bin/ld.bfd linker/ld
```

Comment:
?

Reply:
GOLD doesn't work on PIE-x86, there are a number of Debian (and probably Ubuntu too) bug reports for it.

Reply:
LDC defaults to gold on Linux for a while, incl. linking 32-bit stuff, without any problems I would remember.

Reply:
For example: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=846921
Also, there's a dmd issue: https://issues.dlang.org/show_bug.cgi?id=18798
There's an acknowledged issue with gold in binutils: https://sourceware.org/bugzilla/show_bug.cgi?id=18296

Reply:
And if it really doesn't work for DMD somehow, I'd suggest skipping this block in that case only - `if [[ "$OS_NAME" == "linux" && "$MODEL" != "32" ]]`.

Reply:
> LDC defaults to gold on Linux for a while, incl. linking 32-bit stuff, without any problems I would remember.

FYI I had to change it back to bfd on Alpine as well.